onert-micro/onert-micro/include/pal/common/PALGRUWeightGrad.h

```diff
@@ -62,6 +62,9 @@ void calculateGRUWeightGrads(
     float tanh_grad_value;
     {
       float tanh = std::tanh(tanh_data[i]);
+      if(tanh_data[i] > 19) {tanh = 1;}
+      else if(tanh_data[i] < -19) {tanh = -1;}
+
```

Comment:
@chunseoklee 
Sorry for 
but m.b. it would be better to avoid calculation of `tanh` for large values as was done in `PALGRUCommon.h`, like this 
```
float tanh = 0.f;
 if(tanh_data[i] > 19) {tanh = 1;}
      else if(tanh_data[i] < -19) {tanh = -1;}
else{
tanh = std::tanh(tanh_data[i]);
}
```
Otherwise LGTM!

Reply:
I forgot format-check.